content/en/metrics/guide/micrometer.md

```diff
@@ -0,0 +1,41 @@
+---
+title: Send metrics with Micrometer
+kind: Guide
+further_reading:
+- link: "https://micrometer.io/docs/registry/otlp"
+  tag: "Micrometer"
+  text: "Micrometer Prometheus"
+- link: "https://micrometer.io/docs/registry/prometheus"
```

Comment:
```suggestion
- link: "https://micrometer.io/docs/registry/prometheus"
  tag: "Micrometer"
  text: "Micrometer Prometheus"
- link: "https://micrometer.io/docs/registry/otlp"
```

Reply:
Not sure what the intended order is, but either the link or the text needs to be swapped for these two further reading resources

Reply:
Thanks for catching this @buraizu!